nielsen/src/nielsen/Types.ts

```diff
@@ -23,3 +220,34 @@ export type AdMetadata = {
     type: AdType;
     assetid: any; // TODO string? or can be anything?
 } & { [key: string]: string };
+
+export type DCRAdMetadataCZ = AdMetadata & {
+    /*
+     * An item in the CMS tag reserved for an identifier enabling the connection of an advertisement description from the RTVK system, similarly to PEM TV data.
+     */
+    nol_c4: string;
+    /*
+     * More detailed categorization of video content
+     */
+    nol_c5: string;
+    /*
+     * Ad type (same value as in the "type" item)
+     */
+    nol_c6: string;
+    /*
+     * Ad description. Possible use for cases where the AKA code is not available. RTB - designation of the advertising supplier (e.g. if there is no AKA code or more detailed description of the advertisement).
+     */
+    title?: string;
+    /*
+     * Length of broadcast ad in seconds. (So that the length indicator is available even in cases where the AKA code is not available.)
+     */
+    length: string;
+};
+
+/*
+ * Countries for which (1) Nielsen provides DCR Browser SDKs and (2) the corresponding SDK was tested with this integration.
```

Comment:
Is the country only for DCR? I thought DTVR also has some country specific differences.
It also seems that at some places we associate US with DTVR at the moment.